src/ld53/game-ld53.ts

```diff
@@ -193,6 +197,24 @@ export async function initLD53(em: EntityManager, hosting: boolean) {
   em.ensureComponentOn(sunlight, PositionDef, V(50, 300, 10));
   em.ensureComponentOn(sunlight, RenderableConstructDef, res.assets.ball.proto);
 
+  // pirate test
+  const PirateDef = EM.defineComponent("pirate", () => true);
+  const pirate = em.new();
+  em.ensureComponentOn(
+    pirate,
+    RiggedRenderableConstructDef,
+    res.assets.pirate.mesh as RiggedMesh
```

Comment:
instead of this cast, you could just have an:
```
assert(isRiggedMesh(res.assets.pirate.mesh))
```
above. Ts will (should) narrow appropriately